aesara/scalar/basic.py

```diff
@@ -2133,10 +2133,7 @@ def grad(self, inputs, g_output):
         return [inp.zeros_like(dtype=config.floatX) for inp in inputs]
 
 
-int_div = IntDiv(upcast_out, name="int_div")
-
-
-floor_div = int_div
+floor_divide = IntDiv(upcast_out, name="floor_divide")
```

Comment:
Since we are renaming everything, should we also rename the class `IntDiv`, I thing that will make our code consistence

Reply:
It's a good idea